python/lsst/ts/standardscripts/maintel/slew.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["Slew"]
```

Comment:
The original name slew_icrs was to indicate that the telescope slewed to a position in ICRS coordinates, as opposed to slewing to a position in el/az. We need both. Please keep ICRS in the name or make this general enough to support both kinds of targets.

Reply:
So, this script works with ICR coordinates and object name so ICRS would not be a good representation. Maybe I can call this "TrackTarget" instead?